docs/3.0/guides/acts_as_tenant_integration.md

```diff
@@ -0,0 +1,152 @@
+# Acts As Tenant Integration
+
+Recipe [contributed](https://github.com/avo-hq/docs.avohq.io/pull/218) by [SahSantoshh](https://github.com/sahsantoshh).
+
+There are different ways to achieve multi-tenancy in an application.
+We already have a doc which describes about [Multitenancy](../multitenancy.md) with Avo.
+Here we will deep dive in integrating [Acts As Tenant](https://github.com/ErwinM/acts_as_tenant) which supports row-level multitenancy with Avo.
+In this implementation we will be setting tenant to subdomain.
+
+:::info
+Check out the [acts_as_tenant](https://github.com/ErwinM/acts_as_tenant) documentation for reference.
+:::
+
+## Installation
+___
+
+To use it, add it to your Gemfile:
+
+:::code-group
+```ruby [Gemfile]{3}
+gem 'acts_as_tenant'
+```
+:::
```

Comment:
If it's just one file, you can ommit the code-group statement.

```suggestion
```ruby
gem 'acts_as_tenant'
```
```